pynumad/io/yaml_to_blade.py

```diff
@@ -96,8 +94,8 @@ def yaml_to_blade(blade, filename: str, write_airfoils: bool = False):
     
     # Leading and Trailing Edge bands are constants in millimeters
 
-    blade.leband = np.array(blade_internal_structure['layers'][I_LE]['width']['values'])*1000 / 2
```

Comment:
Kirk, we need to keep these as arrays instead of constants.

Reply:
Okay I wasn't sure if that was apart of the changes Evan made. I'll pull it back in.